packages/help-center/src/components/help-center-feedback-form.tsx

```diff
@@ -0,0 +1,65 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useI18n } from '@wordpress/react-i18n';
+import React, { useState } from 'react';
+import { ThumbsDownIcon, ThumbsUpIcon } from '../icons/thumbs';
+import './help-center-feedback-form.scss';
+interface HelpCenterFeedbackFormProps {
+	postId: number;
+	blogId?: string | null;
+}
+const HelpCenterFeedbackForm = ( { postId, blogId }: HelpCenterFeedbackFormProps ) => {
+	const { __ } = useI18n();
+	const [ startedFeedback, setStartedFeedback ] = useState< boolean | null >( null );
+	const [ answerValue, setAnswerValue ] = useState< number | null >( null );
+
+	const handleFeedbackClick = ( value: number ) => {
+		setStartedFeedback( true );
+		setAnswerValue( value );
+
+		recordTracksEvent( `calypso_inlinehelp_article_feedback_click`, {
+			did_the_article_help: value === 1 ? 'yes' : 'no',
+			post_id: postId,
+		} );
+	};
+
+	const FeedbackButtons = () => (
+		<>
+			<p>{ __( 'Did you find the answer to your question?' ) }</p>
+			<div className="help-center-feedback-form__buttons">
+				<button
+					// 1 is used as `yes` in crowdsignal as well, do not change
+					onClick={ () => handleFeedbackClick( 1 ) }
+				>
+					{ __( 'Yes' ) } <ThumbsUpIcon />
+				</button>
+				<button
+					// 2 is used as `no` in crowdsignal as well, do no change
+					onClick={ () => handleFeedbackClick( 2 ) }
+				>
+					{ __( 'No' ) } <ThumbsDownIcon />
+				</button>
+			</div>
+		</>
+	);
+
+	const FeedbackTextArea = () => (
+		<>
+			<p>{ __( 'How we can improve?' ) }</p>
+			<iframe
```

Comment:
Loads `crowdsignal` survey, the submit happens in this tool and not in HelpCenter.